docs/organizations/managing-security-and-risk.md

```diff
@@ -0,0 +1,111 @@
+
+Under Security and Risk Management, you have the following pages to help you monitor your repositories:
+
+-   [Dashboard](#dashboard)
+-   [Item list](#item-list)
+
+## Dashboard
+
+The Security and Risk Management dashboard provides a general overview of security items, based on their status.
+
+To access the dashboard, select an organization from the top navigation bar and select **Security and risk** on the left navigation sidebar.
+
+The dashboard includes five areas:
+
+-   **Total:** all open items
+-   **Due soon:** open items within 15 days of their deadline
+-   **Overdue:** open items with a missed deadline
+-   **Closed on time:** items closed before the deadline
+-   **Closed late:** items closed after the deadline
+
+Each of these areas displays a count of matching items and a **Review** button to access a filtered list displaying the matching items.
+
+![Security and Risk Management dashboard](images/placeholder.png)
+
+## Item list
+
+The Security and Risk Management item list displays a list of items, filtered by status and ordered by days to deadline, with overdue items first, followed by items closest to the deadline.
+
+To access the item list, access the [dashboard](#dashboard) and click the **Review** button under the area of interest, based on the desired filtering.
+
+To find out more about an item, click its **Details** column. This redirects you to a filtered view of the **Repository issues** page, showing only the item of interest.
+
+![Security and Risk Management item list](images/placeholder.png)
+
+## Opening and closing items
+
+Codacy automatically opens a new item when it detects a new Security issue on the default branch of a repository:
+
+-   [Every new item is assigned a priority](#item-priorities-and-deadlines) based on the severity of the underlying issue. Each priority also defines a deadline to fix the underlying issue.
+
+-   [Every new item is assigned a status](#item-statuses) based on the number of days left before the deadline to fix the underlying issue. As deadlines are approached, met, or missed, items transition through different statuses.
+
+Codacy automatically closes an item when it stops detecting the underlying issue, when [the underlying issue is ignored](../repositories/issues.md#ignoring-and-managing-issues), or when the [tool that found the underlying issue is disabled](../repositories-configure/configuring-code-patterns.md).
+
+For example, Codacy automatically opens an item with Critical priority upon detecting a new Security issue of Critical severity on the default branch of a repository. If the Security issue is fixed before the deadline, the status of the item changes to Closed on time. If the issue isn't fixed before the deadline, the status of the item changes to Overdue.
+
+!!! important
+    To automatically open and close items, Codacy must detect when the underlying Security issues are introduced and fixed. To make sure that Codacy detects the Security issues correctly:
```

Comment:
do you know if this excludes "possible" fixed issues?

Reply:
@mrfyda Can you clarify? We mention [earlier](https://github.com/codacy/docs/blob/45e1fe2f0713e21035b56175e1047528c8e2ff7a/docs/organizations/managing-security-and-risk.md?plain=1#L49) that items are closed also if the issue is ignored or if the tool is disabled. Is this related?

Reply:
I think that @mrfyda is talking about "[possible issues](https://docs.codacy.com/repositories/pull-requests/#possible-issues)" - basically, issues introduced or fixed in lines that aren't directly changed by a commit or pull request.

Reply:
exactly! thanks for clarifying

the scenario where I see a potential problem is:

1. Codacy identifies a new security issue
2. the issue is reported in the dashboard
3. in a subsequent commit, Codacy considers the issue fixed (but as a "possible" fixed issue)
4. the issue continues to be reported in the dashboard endlessly

Reply:
Thanks @prcr!

@machadoit or @lventura-codacy any caveats on possible issues?

Reply:
Hey, so the main ideia is that we don't rely on any new / fixed / possible new / possible fix that you see on the UI. We rely on 'snapshots' of issues at a given time of a repo. 

We went for a more robust solution to avoid the problems that you mention, and also if people just add / remove tools that make issues appear outside of diffs of analysed commits.

Let me know if this is enough to clarify the question, but we are not expecting any problems with the scenario that you mentioned